components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -34,14 +34,6 @@ type Props = {
     trialJustStarted?: boolean;
 }
 
-const ConsolePages = {
```

Comment:
moved this to constants, since in the trial benefits modal we have something similar to the learn more links and those point to system console pages, I reused/copied some of the logic there and then realized that it makes sense to keep those system console urls in a central place.

Reply:
good catch